data/emission_factors.csv

```diff
@@ -1,100 +1,307 @@
+235,global,vehicle,car,mobitool,Personenkraftwagen,P.km,small,,diesel,kg/P.km,0.1109,
+236,global,vehicle,car,mobitool,Personenkraftwagen,P.km,medium,,diesel,kg/P.km,0.1322,
+237,global,vehicle,car,mobitool,Personenkraftwagen,P.km,large,,diesel,kg/P.km,0.1757,
+238,global,public_transport,train,mobitool,Eisenbahn,P.km,,long-distance,,kg/P.km,0.0068,
+239,global,public_transport,train,mobitool,Eisenbahn,P.km,,local,,kg/P.km,0.008,
+240,global,public_transport,train,mobitool,Eisenbahn,P.km,,average,,kg/P.km,0.007,
+241,global,public_transport,train,mobitool,Strassenbahn,P.km,,local,electric,kg/P.km,0.0428,
+242,DE,public_transport,train,mobitool,Bahn Deutschland,P.km,,average,,kg/P.km,0.0408,
+243,FR,public_transport,train,mobitool,Bahn Frankreich,P.km,,average,,kg/P.km,0.0125,
+244,IT,public_transport,train,mobitool,Bahn Italien,P.km,,average,,kg/P.km,0.0747,
+245,AT,public_transport,train,mobitool,Bahn Österreich,P.km,,average,,kg/P.km,0.0151,
+246,global,vehicle,car,mobitool,Personenkraftwagen,P.km,large,,hybrid,kg/P.km,0.1785,
+247,global,vehicle,car,mobitool,Personenkraftwagen,P.km,small,,hybrid,kg/P.km,0.1182,
+248,global,vehicle,car,mobitool,Personenkraftwagen,P.km,medium,,hybrid,kg/P.km,0.1397,
+249,global,vehicle,car,mobitool,Personenkraftwagen,P.km,average,,hybrid,kg/P.km,0.1397,
+250,global,vehicle,car,mobitool,Personenkraftwagen,P.km,average,,plug-in_hybrid,kg/P.km,0.1278,
+251,global,vehicle,car,mobitool,Personenkraftwagen,P.km,small,,plug-in_hybrid,kg/P.km,0.0799,
+252,global,vehicle,car,mobitool,Personenkraftwagen,P.km,medium,,plug-in_hybrid,kg/P.km,0.1278,
+253,global,vehicle,car,mobitool,Personenkraftwagen,P.km,large,,plug-in_hybrid,kg/P.km,0.1718,
+254,global,vehicle,car,mobitool,Personenkraftwagen,P.km,average,,electric,kg/P.km,0.0898,
+255,global,vehicle,car,mobitool,Personenkraftwagen,P.km,small,,electric,kg/P.km,0.0599,
+256,global,vehicle,car,mobitool,Personenkraftwagen,P.km,medium,,electric,kg/P.km,0.0697,
+257,global,vehicle,car,mobitool,Personenkraftwagen,P.km,large,,electric,kg/P.km,0.091,
+258,global,public transport,plane,mobitool,Flugzeug,P.km,,average,kerosine,kg/P.km,0.263,average
+259,global,public transport,plane,mobitool,Flugzeug,P.km,,short-haul,kerosine,kg/P.km,0.3192,average
+260,global,public transport,plane,mobitool,Flugzeug,P.km,,short-haul,kerosine,kg/P.km,0.2918,economy_class
+261,global,public transport,plane,mobitool,Flugzeug,P.km,,short-haul,kerosine,kg/P.km,0.4488,business_class
+262,global,public transport,plane,mobitool,Flugzeug,P.km,,long-haul,kerosine,kg/P.km,0.2372,average
+263,global,public transport,plane,mobitool,Flugzeug,P.km,,long-haul,kerosine,kg/P.km,0.1895,economy_class
+264,global,public transport,plane,mobitool,Flugzeug,P.km,,long-haul,kerosine,kg/P.km,0.3914,business_class
+265,global,public transport,plane,mobitool,Flugzeug,P.km,,long-haul,kerosine,kg/P.km,0.6031,first_class
+266,global,public transport,ferry,mobitool,Personenschiff,P.km,,,diesel,kg/P.km,0.1613,

```

Comment:
Are we not distinguishing whether you go on a ferry with your car or not?
We had 0.01874 (foot-passenger) vs. 0.12952 (car-passenger) in the UK emission factors here before..